knative-operator/pkg/webhook/knativeserving/webhook_validating.go

```diff
@@ -64,6 +64,7 @@ func (v *KnativeServingValidator) validate(ctx context.Context, ks *servingv1alp
 	stages := []func(context.Context, *servingv1alpha1.KnativeServing) (bool, string, error){
 		v.validateNamespace,
```

Comment:
Do we need to validate the name too or do we not care about the name currently and just care about it being in the "knative-serving" namespace and alone?

Reply:
Answered in Slack, this seems fine.